src/main/java/com/example/enterpriseapplication/RecipeManager.java

```diff
@@ -0,0 +1,50 @@
+package com.example.enterpriseapplication;
+
+import com.example.enterpriseapplication.dto.Recipe;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+
+import java.io.*;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+
+public class RecipeManager {
```

Comment:
As mentioned in another comment, I extracted code from another file and gave it its own.